Bottom: 669933672237fc34a70e6596aa8a38bde88c522b
Top:    f1e9c464e56db967919df44c3e92faa5364d3657
Author: Ben Greear <greearb@candelatech.com>
Date:   2015-08-13 13:03:44 -0700

32-bit kernel config for 4.2 kernel.


---

diff --git a/configs/ct-x32.cfg b/configs/ct-x32.cfg
index 7b0cf20..6755338 100644
--- a/configs/ct-x32.cfg
+++ b/configs/ct-x32.cfg
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.2.0-rc5 Kernel Configuration
+# Linux/x86 4.2.0-rc6 Kernel Configuration
 #
 # CONFIG_64BIT is not set
 CONFIG_X86_32=y
@@ -2120,7 +2120,6 @@ CONFIG_NETCONSOLE=m
 CONFIG_NETCONSOLE_DYNAMIC=y
 CONFIG_NETPOLL=y
 CONFIG_NET_POLL_CONTROLLER=y
-CONFIG_NTB_NETDEV=m
 CONFIG_TUN=m
 # CONFIG_TUN_VNET_CROSS_LE is not set
 CONFIG_VETH=m
@@ -3014,7 +3013,7 @@ CONFIG_TOUCHSCREEN_TOUCHWIN=m
 CONFIG_TOUCHSCREEN_TI_AM335X_TSC=m
 CONFIG_TOUCHSCREEN_UCB1400=m
 CONFIG_TOUCHSCREEN_PIXCIR=m
-CONFIG_TOUCHSCREEN_WDT87XX_I2C=m
+# CONFIG_TOUCHSCREEN_WDT87XX_I2C is not set
 CONFIG_TOUCHSCREEN_WM831X=m
 # CONFIG_TOUCHSCREEN_WM97XX is not set
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
@@ -3044,7 +3043,7 @@ CONFIG_TOUCHSCREEN_TSC2007=m
 CONFIG_TOUCHSCREEN_PCAP=m
 CONFIG_TOUCHSCREEN_ST1232=m
 CONFIG_TOUCHSCREEN_SUR40=m
-CONFIG_TOUCHSCREEN_SX8654=m
+# CONFIG_TOUCHSCREEN_SX8654 is not set
 # CONFIG_TOUCHSCREEN_TPS6507X is not set
 CONFIG_TOUCHSCREEN_ZFORCE=m
 CONFIG_INPUT_MISC=y
@@ -3055,7 +3054,6 @@ CONFIG_INPUT_88PM80X_ONKEY=m
 CONFIG_INPUT_E3X0_BUTTON=m
 CONFIG_INPUT_PCSPKR=m
 # CONFIG_INPUT_MAX77693_HAPTIC is not set
-CONFIG_INPUT_MAX77843_HAPTIC=m
 CONFIG_INPUT_MAX8925_ONKEY=m
 CONFIG_INPUT_MAX8997_HAPTIC=m
 CONFIG_INPUT_MC13783_PWRBUTTON=m
@@ -3088,7 +3086,7 @@ CONFIG_INPUT_PWM_BEEPER=m
 CONFIG_INPUT_GPIO_ROTARY_ENCODER=m
 CONFIG_INPUT_DA9052_ONKEY=m
 CONFIG_INPUT_DA9055_ONKEY=m
-CONFIG_INPUT_DA9063_ONKEY=m
+# CONFIG_INPUT_DA9063_ONKEY is not set
 CONFIG_INPUT_WM831X_ON=m
 CONFIG_INPUT_PCAP=m
 # CONFIG_INPUT_ADXL34X is not set
@@ -3099,7 +3097,7 @@ CONFIG_INPUT_XEN_KBDDEV_FRONTEND=y
 CONFIG_INPUT_IDEAPAD_SLIDEBAR=m
 CONFIG_INPUT_SOC_BUTTON_ARRAY=m
 # CONFIG_INPUT_DRV260X_HAPTICS is not set
-CONFIG_INPUT_DRV2665_HAPTICS=m
+# CONFIG_INPUT_DRV2665_HAPTICS is not set
 # CONFIG_INPUT_DRV2667_HAPTICS is not set
 
 #
@@ -3187,7 +3185,7 @@ CONFIG_SERIAL_SCCNXP=m
 CONFIG_SERIAL_SC16IS7XX_CORE=m
 CONFIG_SERIAL_SC16IS7XX=m
 CONFIG_SERIAL_SC16IS7XX_I2C=y
-CONFIG_SERIAL_SC16IS7XX_SPI=y
+# CONFIG_SERIAL_SC16IS7XX_SPI is not set
 # CONFIG_SERIAL_TIMBERDALE is not set
 # CONFIG_SERIAL_ALTERA_JTAGUART is not set
 # CONFIG_SERIAL_ALTERA_UART is not set
@@ -3376,7 +3374,7 @@ CONFIG_SPI_XILINX=m
 CONFIG_SPI_ZYNQMP_GQSPI=m
 CONFIG_SPI_DESIGNWARE=m
 CONFIG_SPI_DW_PCI=m
-CONFIG_SPI_DW_MID_DMA=y
+# CONFIG_SPI_DW_MID_DMA is not set
 CONFIG_SPI_DW_MMIO=m
 
 #
@@ -3767,7 +3765,7 @@ CONFIG_SENSORS_ACPI_POWER=m
 CONFIG_SENSORS_ATK0110=m
 CONFIG_THERMAL=y
 CONFIG_THERMAL_HWMON=y
-CONFIG_THERMAL_WRITABLE_TRIPS=y
+# CONFIG_THERMAL_WRITABLE_TRIPS is not set
 CONFIG_THERMAL_DEFAULT_GOV_STEP_WISE=y
 # CONFIG_THERMAL_DEFAULT_GOV_FAIR_SHARE is not set
 # CONFIG_THERMAL_DEFAULT_GOV_USER_SPACE is not set
@@ -3782,7 +3780,7 @@ CONFIG_INTEL_POWERCLAMP=m
 CONFIG_X86_PKG_TEMP_THERMAL=m
 CONFIG_INTEL_SOC_DTS_IOSF_CORE=m
 CONFIG_INTEL_SOC_DTS_THERMAL=m
-CONFIG_INTEL_QUARK_DTS_THERMAL=m
+CONFIG_INTEL_QUARK_DTS_THERMAL=y
 # CONFIG_INT340X_THERMAL is not set
 
 #
@@ -3905,6 +3903,7 @@ CONFIG_MFD_BCM590XX=m
 CONFIG_MFD_AXP20X=y
 CONFIG_MFD_CROS_EC=m
 CONFIG_MFD_CROS_EC_I2C=m
+# CONFIG_MFD_CROS_EC_SPI is not set
 CONFIG_PMIC_DA903X=y
 CONFIG_PMIC_DA9052=y
 CONFIG_MFD_DA9052_SPI=y
@@ -3930,12 +3929,12 @@ CONFIG_MFD_88PM805=m
 CONFIG_MFD_88PM860X=y
 CONFIG_MFD_MAX14577=y
 CONFIG_MFD_MAX77693=y
-CONFIG_MFD_MAX77843=y
+# CONFIG_MFD_MAX77843 is not set
 CONFIG_MFD_MAX8907=y
 CONFIG_MFD_MAX8925=y
 CONFIG_MFD_MAX8997=y
 CONFIG_MFD_MAX8998=y
-CONFIG_MFD_MT6397=m
+# CONFIG_MFD_MT6397 is not set
 # CONFIG_MFD_MENF21BMC is not set
 CONFIG_EZX_PCAP=y
 CONFIG_MFD_VIPERBOARD=m
@@ -3952,7 +3951,7 @@ CONFIG_MFD_SEC_CORE=y
 CONFIG_MFD_SI476X_CORE=m
 CONFIG_MFD_SM501=m
 CONFIG_MFD_SM501_GPIO=y
-CONFIG_MFD_SKY81452=m
+# CONFIG_MFD_SKY81452 is not set
 CONFIG_MFD_SMSC=y
 # CONFIG_ABX500_CORE is not set
 CONFIG_MFD_STA2X11=y
@@ -4035,11 +4034,9 @@ CONFIG_REGULATOR_MAX8973=m
 CONFIG_REGULATOR_MAX8997=m
 CONFIG_REGULATOR_MAX8998=m
 CONFIG_REGULATOR_MAX77693=m
-CONFIG_REGULATOR_MAX77843=m
 CONFIG_REGULATOR_MC13XXX_CORE=m
 CONFIG_REGULATOR_MC13783=m
 CONFIG_REGULATOR_MC13892=m
-CONFIG_REGULATOR_MT6397=m
 CONFIG_REGULATOR_PALMAS=m
 CONFIG_REGULATOR_PCAP=m
 CONFIG_REGULATOR_PFUZE100=m
@@ -4050,7 +4047,6 @@ CONFIG_REGULATOR_RT5033=m
 CONFIG_REGULATOR_S2MPA01=m
 CONFIG_REGULATOR_S2MPS11=m
 CONFIG_REGULATOR_S5M8767=m
-CONFIG_REGULATOR_SKY81452=m
 CONFIG_REGULATOR_TPS51632=m
 CONFIG_REGULATOR_TPS62360=m
 CONFIG_REGULATOR_TPS65023=m
@@ -4087,7 +4083,7 @@ CONFIG_VIDEO_V4L2=m
 # CONFIG_VIDEO_PCI_SKELETON is not set
 CONFIG_VIDEO_TUNER=m
 CONFIG_V4L2_MEM2MEM_DEV=m
-CONFIG_V4L2_FLASH_LED_CLASS=m
+# CONFIG_V4L2_FLASH_LED_CLASS is not set
 CONFIG_VIDEOBUF_GEN=m
 CONFIG_VIDEOBUF_DMA_SG=m
 CONFIG_VIDEOBUF_VMALLOC=m
@@ -4730,9 +4726,7 @@ CONFIG_DRM_I2C_NXP_TDA998X=m
 CONFIG_DRM_RADEON=m
 # CONFIG_DRM_RADEON_USERPTR is not set
 # CONFIG_DRM_RADEON_UMS is not set
-CONFIG_DRM_AMDGPU=m
-CONFIG_DRM_AMDGPU_CIK=y
-CONFIG_DRM_AMDGPU_USERPTR=y
+# CONFIG_DRM_AMDGPU is not set
 CONFIG_DRM_NOUVEAU=m
 CONFIG_NOUVEAU_DEBUG=5
 CONFIG_NOUVEAU_DEBUG_DEFAULT=3
@@ -4745,7 +4739,7 @@ CONFIG_DRM_I915_PRELIMINARY_HW_SUPPORT=y
 # CONFIG_DRM_SIS is not set
 CONFIG_DRM_VIA=m
 # CONFIG_DRM_SAVAGE is not set
-CONFIG_DRM_VGEM=m
+# CONFIG_DRM_VGEM is not set
 CONFIG_DRM_VMWGFX=m
 # CONFIG_DRM_VMWGFX_FBCON is not set
 CONFIG_DRM_GMA500=m
@@ -4758,7 +4752,7 @@ CONFIG_DRM_MGAG200=m
 CONFIG_DRM_CIRRUS_QEMU=m
 CONFIG_DRM_QXL=m
 CONFIG_DRM_BOCHS=m
-CONFIG_DRM_VIRTIO_GPU=m
+# CONFIG_DRM_VIRTIO_GPU is not set
 CONFIG_DRM_PANEL=y
 
 #
@@ -4883,7 +4877,6 @@ CONFIG_BACKLIGHT_LP855X=m
 CONFIG_BACKLIGHT_LP8788=m
 CONFIG_BACKLIGHT_OT200=m
 CONFIG_BACKLIGHT_PANDORA=m
-CONFIG_BACKLIGHT_SKY81452=m
 CONFIG_BACKLIGHT_TPS65217=m
 CONFIG_BACKLIGHT_AS3711=m
 CONFIG_BACKLIGHT_GPIO=m
@@ -5181,7 +5174,7 @@ CONFIG_HID_ZEROPLUS=m
 CONFIG_ZEROPLUS_FF=y
 CONFIG_HID_ZYDACRON=m
 CONFIG_HID_SENSOR_HUB=m
-CONFIG_HID_SENSOR_CUSTOM_SENSOR=m
+# CONFIG_HID_SENSOR_CUSTOM_SENSOR is not set
 
 #
 # USB HID support
@@ -5721,7 +5714,6 @@ CONFIG_RTC_DRV_WM8350=m
 #
 CONFIG_RTC_DRV_PCAP=m
 CONFIG_RTC_DRV_MC13XXX=m
-CONFIG_RTC_DRV_MT6397=m
 
 #
 # HID Sensor RTC drivers
@@ -6073,8 +6065,8 @@ CONFIG_INTEL_PMC_IPC=m
 CONFIG_CHROME_PLATFORMS=y
 CONFIG_CHROMEOS_LAPTOP=m
 CONFIG_CHROMEOS_PSTORE=m
-CONFIG_CROS_EC_CHARDEV=m
-CONFIG_CROS_EC_LPC=m
+# CONFIG_CROS_EC_CHARDEV is not set
+# CONFIG_CROS_EC_LPC is not set
 CONFIG_CROS_EC_PROTO=y
 CONFIG_CLKDEV_LOOKUP=y
 CONFIG_HAVE_CLK_PREPARE=y
@@ -6163,7 +6155,6 @@ CONFIG_EXTCON_AXP288=m
 CONFIG_EXTCON_GPIO=m
 CONFIG_EXTCON_MAX14577=m
 CONFIG_EXTCON_MAX77693=m
-CONFIG_EXTCON_MAX77843=m
 CONFIG_EXTCON_MAX8997=m
 CONFIG_EXTCON_PALMAS=m
 # CONFIG_EXTCON_RT8973A is not set
@@ -6374,8 +6365,8 @@ CONFIG_HID_SENSOR_PRESS=m
 CONFIG_MPL115=m
 CONFIG_MPL3115=m
 CONFIG_MS5611=m
-CONFIG_MS5611_I2C=m
-CONFIG_MS5611_SPI=m
+# CONFIG_MS5611_I2C is not set
+# CONFIG_MS5611_SPI is not set
 CONFIG_IIO_ST_PRESS=m
 CONFIG_IIO_ST_PRESS_I2C=m
 CONFIG_IIO_ST_PRESS_SPI=m
@@ -6397,9 +6388,9 @@ CONFIG_SX9500=m
 CONFIG_MLX90614=m
 CONFIG_TMP006=m
 CONFIG_NTB=m
-CONFIG_NTB_PINGPONG=m
-CONFIG_NTB_TOOL=m
-CONFIG_NTB_TRANSPORT=m
+# CONFIG_NTB_PINGPONG is not set
+# CONFIG_NTB_TOOL is not set
+# CONFIG_NTB_TRANSPORT is not set
 # CONFIG_VME_BUS is not set
 CONFIG_PWM=y
 CONFIG_PWM_SYSFS=y
@@ -6484,8 +6475,7 @@ CONFIG_EXT4_FS=y
 CONFIG_EXT4_USE_FOR_EXT23=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
-CONFIG_EXT4_ENCRYPTION=m
-CONFIG_EXT4_FS_ENCRYPTION=y
+# CONFIG_EXT4_ENCRYPTION is not set
 # CONFIG_EXT4_DEBUG is not set
 CONFIG_JBD2=y
 # CONFIG_JBD2_DEBUG is not set
@@ -7004,8 +6994,8 @@ CONFIG_OPTIMIZE_INLINING=y
 # CONFIG_DEBUG_NMI_SELFTEST is not set
 # CONFIG_DEBUG_IMR_SELFTEST is not set
 # CONFIG_X86_DEBUG_STATIC_CPU_HAS is not set
-CONFIG_X86_DEBUG_FPU=y
-# CONFIG_PUNIT_ATOM_DEBUG is not set
+# CONFIG_X86_DEBUG_FPU is not set
+CONFIG_PUNIT_ATOM_DEBUG=m
 
 #
 # Security options
@@ -7086,13 +7076,13 @@ CONFIG_CRYPTO_MCRYPTD=y
 CONFIG_CRYPTO_AUTHENC=y
 CONFIG_CRYPTO_TEST=m
 CONFIG_CRYPTO_ABLK_HELPER=y
-CONFIG_CRYPTO_GLUE_HELPER_X86=m
+CONFIG_CRYPTO_GLUE_HELPER_X86=y
 
 #
 # Authenticated Encryption with Associated Data
 #
-CONFIG_CRYPTO_CCM=m
-CONFIG_CRYPTO_GCM=m
+CONFIG_CRYPTO_CCM=y
+CONFIG_CRYPTO_GCM=y
 CONFIG_CRYPTO_CHACHA20POLY1305=y
 CONFIG_CRYPTO_SEQIV=y
 CONFIG_CRYPTO_ECHAINIV=y
@@ -7102,10 +7092,10 @@ CONFIG_CRYPTO_ECHAINIV=y
 #
 CONFIG_CRYPTO_CBC=y
 CONFIG_CRYPTO_CTR=y
-CONFIG_CRYPTO_CTS=m
+CONFIG_CRYPTO_CTS=y
 CONFIG_CRYPTO_ECB=y
 CONFIG_CRYPTO_LRW=y
-CONFIG_CRYPTO_PCBC=m
+CONFIG_CRYPTO_PCBC=y
 CONFIG_CRYPTO_XTS=y
 
 #
@@ -7113,31 +7103,31 @@ CONFIG_CRYPTO_XTS=y
 #
 CONFIG_CRYPTO_CMAC=y
 CONFIG_CRYPTO_HMAC=y
-CONFIG_CRYPTO_XCBC=m
-CONFIG_CRYPTO_VMAC=m
+CONFIG_CRYPTO_XCBC=y
+CONFIG_CRYPTO_VMAC=y
 
 #
 # Digest
 #
 CONFIG_CRYPTO_CRC32C=y
-CONFIG_CRYPTO_CRC32C_INTEL=m
+CONFIG_CRYPTO_CRC32C_INTEL=y
 CONFIG_CRYPTO_CRC32=y
 CONFIG_CRYPTO_CRC32_PCLMUL=y
 CONFIG_CRYPTO_CRCT10DIF=y
-CONFIG_CRYPTO_GHASH=m
+CONFIG_CRYPTO_GHASH=y
 CONFIG_CRYPTO_POLY1305=y
-CONFIG_CRYPTO_MD4=m
+CONFIG_CRYPTO_MD4=y
 CONFIG_CRYPTO_MD5=y
-CONFIG_CRYPTO_MICHAEL_MIC=m
-CONFIG_CRYPTO_RMD128=m
-CONFIG_CRYPTO_RMD160=m
-CONFIG_CRYPTO_RMD256=m
-CONFIG_CRYPTO_RMD320=m
+CONFIG_CRYPTO_MICHAEL_MIC=y
+CONFIG_CRYPTO_RMD128=y
+CONFIG_CRYPTO_RMD160=y
+CONFIG_CRYPTO_RMD256=y
+CONFIG_CRYPTO_RMD320=y
 CONFIG_CRYPTO_SHA1=y
 CONFIG_CRYPTO_SHA256=y
 CONFIG_CRYPTO_SHA512=y
-CONFIG_CRYPTO_TGR192=m
-CONFIG_CRYPTO_WP512=m
+CONFIG_CRYPTO_TGR192=y
+CONFIG_CRYPTO_WP512=y
 
 #
 # Ciphers
@@ -7145,27 +7135,27 @@ CONFIG_CRYPTO_WP512=m
 CONFIG_CRYPTO_AES=y
 CONFIG_CRYPTO_AES_586=y
 CONFIG_CRYPTO_AES_NI_INTEL=y
-CONFIG_CRYPTO_ANUBIS=m
-CONFIG_CRYPTO_ARC4=m
-CONFIG_CRYPTO_BLOWFISH=m
-CONFIG_CRYPTO_BLOWFISH_COMMON=m
-CONFIG_CRYPTO_CAMELLIA=m
-CONFIG_CRYPTO_CAST_COMMON=m
-CONFIG_CRYPTO_CAST5=m
-CONFIG_CRYPTO_CAST6=m
-CONFIG_CRYPTO_DES=m
-CONFIG_CRYPTO_FCRYPT=m
-CONFIG_CRYPTO_KHAZAD=m
-CONFIG_CRYPTO_SALSA20=m
-CONFIG_CRYPTO_SALSA20_586=m
+CONFIG_CRYPTO_ANUBIS=y
+CONFIG_CRYPTO_ARC4=y
+CONFIG_CRYPTO_BLOWFISH=y
+CONFIG_CRYPTO_BLOWFISH_COMMON=y
+CONFIG_CRYPTO_CAMELLIA=y
+CONFIG_CRYPTO_CAST_COMMON=y
+CONFIG_CRYPTO_CAST5=y
+CONFIG_CRYPTO_CAST6=y
+CONFIG_CRYPTO_DES=y
+CONFIG_CRYPTO_FCRYPT=y
+CONFIG_CRYPTO_KHAZAD=y
+CONFIG_CRYPTO_SALSA20=y
+CONFIG_CRYPTO_SALSA20_586=y
 CONFIG_CRYPTO_CHACHA20=y
-CONFIG_CRYPTO_SEED=m
-CONFIG_CRYPTO_SERPENT=m
-CONFIG_CRYPTO_SERPENT_SSE2_586=m
-CONFIG_CRYPTO_TEA=m
-CONFIG_CRYPTO_TWOFISH=m
-CONFIG_CRYPTO_TWOFISH_COMMON=m
-CONFIG_CRYPTO_TWOFISH_586=m
+CONFIG_CRYPTO_SEED=y
+CONFIG_CRYPTO_SERPENT=y
+CONFIG_CRYPTO_SERPENT_SSE2_586=y
+CONFIG_CRYPTO_TEA=y
+CONFIG_CRYPTO_TWOFISH=y
+CONFIG_CRYPTO_TWOFISH_COMMON=y
+CONFIG_CRYPTO_TWOFISH_586=y
 
 #
 # Compression
